masterlist.yaml

```diff
@@ -188,6 +188,11 @@ common:
         text: 'Delete or deactivate %1%.'
       - lang: ru
         text: 'Не забудьте отключить или удалить %1%.'
+  - &redundantPluginX
+    type: warn
+    content:
+      - lang: en
+        text: 'This plugin is redundant when using %1% and should be deactivated or deleted.'
```

Comment:
I've been thinking about this some more:
`&alreadyInX` is fine for cases like 'AOS2_DD Patch.esp'. It's clear that you must delete this patch and not the whole mod and deleting is more appropriate than disabling because these plugins should never have been installed in the first place. The problem are cases like 'RUSTIC SOULGEMS - Unsorted.esp', where the patch is also a replacer plugin (but uses a different name). Here you must disable/delete only the plugin but keep the meshes and textures which is the opposite of what's required in `&AlreadyInUSKP`.

Would it resolve your concern if I used your suggestion for the former case and used a new replacer plugin message for the latter?